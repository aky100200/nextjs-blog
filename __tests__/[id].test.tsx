import renderer from 'react-test-renderer'
import React from 'react'

import Id from '../pages/posts/[id]'

test('IdTest', () => {
  const component = renderer.create(
    <Id
      postData={{
        title: 'test_title',
        date: '2020-01-01',
        contentHtml: '<h1>Hello</h1>',
      }}
    />
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
