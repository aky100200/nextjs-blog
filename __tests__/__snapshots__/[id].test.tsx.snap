// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IdTest 1`] = `
<div>
  <header>
    <a
      href="/"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      <img
        alt="aky100200"
        className="undefined undefined"
        src="/images/profile.jpg"
      />
    </a>
    <h2>
      <a
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        aky100200
      </a>
    </h2>
  </header>
  <main>
    <article>
      <h1>
        test_title
      </h1>
      <div>
        <time
          dateTime="2020-01-01"
        >
          January 1, 2020
        </time>
      </div>
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "<h1>Hello</h1>",
          }
        }
      />
    </article>
  </main>
  <div>
    <a
      href="/"
      onClick={[Function]}
      onMouseEnter={[Function]}
    >
      ‚Üê Back to home
    </a>
  </div>
</div>
`;
